backend/src/apiserver/model/run.go

```diff
@@ -18,6 +18,10 @@ import (
 	"strings"
 )
 
+const (
+	RunTerminatingConditions string = "Terminating"
```

Comment:
The other place that should use this const is in https://github.com/kubeflow/pipelines/blob/7540ba5c3bf003758a5c2f46ccc586a0367c4dee/backend/src/apiserver/storage/run_store.go#L608

Reply:
done